enterprise/server/remote_execution/blockio/blockio.go

```diff
@@ -201,6 +202,8 @@ type Chunk struct {
 // A COWStore can be created either by splitting a file into chunks, or loading
```

Comment:
nit: maybe add a comment that COWStore supports concurrent reads and writes as long as the reads/writes are to different chunks.

Reply:
Done - previously I had only made the blockio code that was used in mergeDiffSnapshot concurrency-safe, so made it concurrency-safe everywhere